examples/gno.land/r/gnoland/faucet/faucet.gno

```diff
@@ -9,60 +9,85 @@ import (
 
 var (
 	// configurable by admin.
-	gAdminAddr       std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gControllerAddr  std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
-	gPerTransferSend             = std.Coins{std.Coin{"ugnot", 1000000}}
-	gInPause                     = false
-	gMessage                     = "# Community Faucet.\n\n"
+	gAdminAddr std.Address = "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5"
```

Comment:
```suggestion
	gAdminAddr std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
```

Reply:
This is, for example, a purpose so that people can use Test1 to play with the faucet contract. We can update the admin address here https://github.com/gnolang/community-faucet-bot

Reply:
OK, good :+1: